test/BasicKeyPairTest.template.ts

```diff
@@ -187,8 +187,8 @@ export const BasicKeyPairTester = (keyPairClass: KeyPairClass, deterministicPriv
             }
 
             // Arrange:
-            const keyPair = keyPairClass.generate();
-            const payload = new Uint8Array([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]);
+            const keyPair: KeyPair = new keyPairClass(deterministicPrivateKey);
+            const payload = new Uint8Array([1, 2, 3, 4, 5, 6, 7, 8, 9, 11]);
```

Comment:
Alright, I think it makes sense because both used different hashers to sign.
In this case, I assume the current payload applies to Symbol, because It took from the old SDK.

How can I find the right payload for nem?